ansible/update_pw_python.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: all
+  tasks:
+    - include: update_pw_python.yml
```

Comment:
This line would seem to be self referential? Is this include including itself, or am I misreading this?